docs/config.rst

```diff
@@ -44,6 +44,21 @@ method::
         SECRET_KEY='...'
     )
 
+.. admonition:: Debug Mode with the ``flask`` Script
```

Comment:
This isn't auto-linking the header (for deep linking) because you are using a code block within the header.